src/components/Features/TransferLog/TransferLog.js

```diff
@@ -66,7 +66,7 @@ export const TransferLog = ({transfer, onCompleteTransferClick, onTxClick}) => {
     <>
       <div className={styles.transferLog}>
         <div className={styles.left}>
-          <CryptoLogo size={CryptoLogoSize.SMALL} symbol={symbol} />
+          <CircleLogo path={`tokens/${symbol}.svg`} size={CircleLogoSize.SMALL} />
```

Comment:
Let's pass only the path (i.e. `tokens/${symbol}`) and let the component itself to add `.svg`?

Reply:
its not as it works in other places. for example https://github.com/starkware-libs/starkgate-frontend/blob/6df9b4dc7b5c1155bc529844da6e72aa37924e20/src/config/wallets.js#L8 (all logoPath prop)

Reply:
It's not the same ad logo icon since the logo icon is not a necessary SVG file.
Seems horrible to add it on every component injection.